### PR TITLE
fix slow db dynamic config bug

### PR DESCRIPTION
**Pull request by zhyyu**
- [ ] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes  [#7169 ](https://github.com/apache/skywalking/issues/7169).
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/CHANGES.md).

If activate dynamic config but not config agent-analyzer.default.slowDBAccessThreshold, the threshold will be changed to a strange value 10000 because of sending wrong delete event. I change the DBLatencyThresholdsAndWatcher, rename settingsString to initialSettingsString and add a new field dynamicSettingsString to avoid wrong delete event.

### CODE REVIEW

File: oap-server/analyzer/agent-analyzer/src/main/java/org/apache/skywalking/oap/server/analyzer/provider/trace/DBLatencyThresholdsAndWatcher.java [hunk 1]
```diff
@@ -23,17 +23,18 @@
...
 import org.apache.skywalking.oap.server.analyzer.module.AnalyzerModule;
 import org.apache.skywalking.oap.server.configuration.api.ConfigChangeWatcher;
-import org.apache.skywalking.oap.server.core.Const;
 import org.apache.skywalking.oap.server.library.module.ModuleProvider;
 
 public class DBLatencyThresholdsAndWatcher extends ConfigChangeWatcher {
     private AtomicReference<Map<String, Integer>> thresholds;
-    private AtomicReference<String> settingsString;
+    private AtomicReference<String> initialSettingsString;
+
```
Comment (wu-sheng) [inline oap-server/analyzer/agent-analyzer/src/main/java/org/apache/skywalking/oap/server/analyzer/provider/trace/DBLatencyThresholdsAndWatcher.java:31]: Why empty line?

File: oap-server/analyzer/agent-analyzer/src/main/java/org/apache/skywalking/oap/server/analyzer/provider/trace/DBLatencyThresholdsAndWatcher.java [hunk 2]
```diff
@@ -23,17 +23,18 @@
 import java.util.concurrent.atomic.AtomicReference;
 import org.apache.skywalking.oap.server.analyzer.module.AnalyzerModule;
 import org.apache.skywalking.oap.server.configuration.api.ConfigChangeWatcher;
-import org.apache.skywalking.oap.server.core.Const;
 import org.apache.skywalking.oap.server.library.module.ModuleProvider;
 
 public class DBLatencyThresholdsAndWatcher extends ConfigChangeWatcher {
     private AtomicReference<Map<String, Integer>> thresholds;
-    private AtomicReference<String> settingsString;
+    private AtomicReference<String> initialSettingsString;
```
Comment (wu-sheng) [inline oap-server/analyzer/agent-analyzer/src/main/java/org/apache/skywalking/oap/server/analyzer/provider/trace/DBLatencyThresholdsAndWatcher.java:30]: `initialSettingsString` should be final, and don't need `AtomicReference`, as it never changed.

**Review by wu-sheng — COMMENTED** [review]
You also should update the changes.md in the root folder.

File: oap-server/analyzer/agent-analyzer/src/main/java/org/apache/skywalking/oap/server/analyzer/provider/trace/DBLatencyThresholdsAndWatcher.java (hunk 1 above)
Comment (zhyyu) [reply oap-server/analyzer/agent-analyzer/src/main/java/org/apache/skywalking/oap/server/analyzer/provider/trace/DBLatencyThresholdsAndWatcher.java]: just my own code style, have deleted the empty line in cr commit.

File: oap-server/analyzer/agent-analyzer/src/main/java/org/apache/skywalking/oap/server/analyzer/provider/trace/DBLatencyThresholdsAndWatcher.java (hunk 2 above)
Comment (zhyyu) [reply oap-server/analyzer/agent-analyzer/src/main/java/org/apache/skywalking/oap/server/analyzer/provider/trace/DBLatencyThresholdsAndWatcher.java]: have changed in cr commit.

**Comment by zhyyu** [comment]
> You also should update the changes.md in the root folder.

changes.md have changed in cr commit.

**Comment by wu-sheng** [comment]
Why this code section still exists?
https://github.com/apache/skywalking/blob/405f0be2645aefce0e2e8ebd5fb8b68c0d87c60f/oap-server/analyzer/agent-analyzer/src/main/java/org/apache/skywalking/oap/server/analyzer/provider/trace/DBLatencyThresholdsAndWatcher.java#L50-L52

**Comment by zhyyu** [comment]
> Why this code section still exists?
> https://github.com/apache/skywalking/blob/405f0be2645aefce0e2e8ebd5fb8b68c0d87c60f/oap-server/analyzer/agent-analyzer/src/main/java/org/apache/skywalking/oap/server/analyzer/provider/trace/DBLatencyThresholdsAndWatcher.java#L50-L52

deleted right now

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**